Python/Product/Debugger/VsPackageMessage.cs

```diff
@@ -12,10 +12,11 @@
  *
  * ***************************************************************************/
 
-namespace Microsoft.PythonTools.DkmDebugger {
+namespace Microsoft.PythonTools.Debugger {

```

Comment:
Ditto. Adding new enum members is okay, but changing the namespace is a breaking change.